mollie/settlements.go

```diff
@@ -13,11 +13,11 @@ import (
 type SettlementsService service
 
 // SettlementStatus describes the status of the settlement
-type SettlementStatus string
+type SettlementStatus
 
 // Available settlement statuses
 const (
-	SettlementStatusOpen    = "string"
+	SettlementStatusOpen    = "open"
 	SettlementStatusPending = "pending"
 	SettlementStatusPaidOut = "paidout"
 	SettlementStatusFailed  = "failed"
```

Comment:
These constants should be typed as `SettlementStatus` string alias.
```suggestion
	SettlementStatusOpen      SettlementStatus = "open"
	SettlementStatusPending  SettlementStatus = "pending"
	SettlementStatusPaidOut  SettlementStatus = "paidout"
	SettlementStatusFailed     SettlementStatus = "failed"
```